src/AddTodoForm.jsx

```diff
@@ -1,22 +1,38 @@
 import * as React from 'react';
-const AddTodoForm = (props) => {
 
-    const handleAddTodo =(event) =>{
-        event.preventDefault();
-        const todoTitle=event.target.title.value
-        console.log(todoTitle)
-        props.onAddTodo(todoTitle)
-        const form=event.target
-        form.reset()
+const AddTodoForm = ({onAddTodo}) => {
+    const [todoTitle, setTodoTitle] = React.useState(''); 
+
+    const handleTitleChange = (event) => {
+        const newTodoTitle = event.target.value;
+        setTodoTitle(newTodoTitle);  
+    };
 
-    }
-    return(
+    const handleAddTodo = (event) => {
+        event.preventDefault();
+        if (todoTitle) {
+            const todoItem = {
+              title: todoTitle,
+              id: Date.now()
+            };
+            onAddTodo(todoItem);
+            setTodoTitle('');  
+        }
+    };
 
-    <form onSubmit={handleAddTodo}>
-        <label htmlFor="todoTitle">Title  </label>
-        <input type="text" id="todoTitle" name="title" />
-        <button type="submit">Add</button>
-    </form>
+    return (
+        <form onSubmit={handleAddTodo}>
+            <label htmlFor="todoTitle">Title</label>
+            <input
+                type="text"
+                value={todoTitle}
+                onChange={handleTitleChange}
```

Comment:
You could also make this simpler and shorter by writing it this way:
`onChange={(e) => setTodoTitle(e.target.value)}`